README.md

```diff
@@ -1,87 +1,69 @@
-need to insert data into the database. There is another make target, `make apisql`,
-which open the `psql` and connect to the database. Thus, you can insert data
-using some `INSERT INTO ...` statements and test the API. ;)
+ATENÇÃO: Quando você precisar reiniciar a API, apenas interrompa o processo da API e execute o `make rerun` novamente. Não é necessário reiniciar o banco de dados.
+Você pode checar toda a documentação interativa da API em  `localhost:8080/docs`. Nessa página, você pode fazer requisições à API diretamente. Mas, para vê-la funcionar, você precisa inserir dados no banco de dados. Há outro comando, `make apisql`, que abre o `psql` e conecta ao banco de dados. Assim, você pode inserir dados usando alguns `INSERT INTO ...` para testar a API  ;)
 
+### Usando o endpoint de ‘sugestões’
 
-### Using suggestion endpoint
+O endpoint de sugestões no Querido Diário é uma forma de coletar feedback dos usuários e usa o serviço do Mailjet para enviar e-mails. É necessário criar um token de acesso em [Mailjet](www.mailjet.com) para executar a aplicação e enviar e-mails  (salve em `config/current.env`).
 
-You need to create a token at [Mailjet](www.mailjet.com) to run
-application and send email (put on `config/current.env`).
+## Testes
 
-## Tests
+Este projeto foi desenvolvido com o paradigma TDD (Test Driven-Development). Isso significa que não há mudanças sem testes. Devemos sempre buscar ter uma cobertura de 100% do código-fonte. Outra maneira de encarar os testes é o seguinte:
 
-The project uses TDD during development. This means that there are no changes 
-without tests. We should always seek 100% source code coverage. Another way to
-think about tests is the following:
+>  “Escreva o teste que o força a escrever o código que você já sabe que quer escrever” 
 
->  "Write the test which forces you to write the code which you already know that you wanna write."
-> By Robert C. Martin (a.k.a. Uncle Bob)
+> Por Robert C. Martin (a.k.a. Uncle Bob), tradução nossa.
 
-To run the tests you can do the following:
+Para executar os testes, faça o seguinte:
 
 ```bash
 make test
 ```
 
-NOTE: You do not need to restart the test database all the times you want to
-run the tests. Once the database is running, you need to run the `make retest` 
-again.
-
-You should do that just the first time you run the tests. After that, you can just
-run `make retest`. Of course, if you remove the database with `make destroydatabse`
-or reboot the machine, you need to start the database again.
-
-If you can to see the code coverage:
+ATENÇÃO: Não é necessário reiniciar a base de teste a cada vez que você quiser executar os testes. Uma vez que o banco de dados esteja em execução, você só precisa rodar o  `make retest` novamente. Claro que, se você remover o banco de dados com `make destroydatabse` ou reiniciar a máquina, você terá que iniciar o banco de dados novamente. 
+Para checar a cobertura do código:
```

Comment:
Aqui repito o comentário que fiz antes sobre o novo parágrafo.

---
### English

Here I repeat the same comment I made about the new paragraph.